pkg/skuba/actions/node/bootstrap/bootstrap.go

```diff
@@ -92,20 +92,39 @@ func Bootstrap(bootstrapConfiguration deployments.BootstrapConfiguration, target
 	}
 
 	fmt.Println("[bootstrap] applying init configuration to node")
-	err = target.Apply(
-		bootstrapConfiguration,
-		"kubernetes.bootstrap.upload-secrets",
-		"kernel.load-modules",
-		"kernel.configure-parameters",
-		"apparmor.start",
-		"cri.start",
-		"kubelet.configure",
-		"kubelet.enable",
-		"kubeadm.init",
-		"psp.deploy",
-		"kured.deploy",
-		"skuba-update.start",
-	)
+	if _, err := os.Stat(skuba.CriDockerDefaultsConfFile()); err == nil {
+		err = target.Apply(
+			bootstrapConfiguration,
+			"kubernetes.bootstrap.upload-secrets",
+			"kernel.load-modules",
+			"kernel.configure-parameters",
+			"apparmor.start",
+			"cri.configure",
```

Comment:
This seems a bit redundant. Maybe you could have a variable `cri_states` that could be

```Go
criStates=[]string{"cri.start"}
if _, err := os.Stat(skuba.CriDockerDefaultsConfFile()); err == nil {
  criStates=append("cri.configure", criStates...)
}
```

and then something like:

```Go
err = target.Apply(
		bootstrapConfiguration,
		"kubernetes.bootstrap.upload-secrets",
		"kernel.load-modules",
		"kernel.configure-parameters",
		"apparmor.start",
		criStates,
		"kubelet.configure",
		"kubelet.enable",
		"kubeadm.init",
		"psp.deploy",
		"kured.deploy",
		"skuba-update.start",
	)
```

Reply:
Let me fix that. Thanks Alvaro :D

Reply:
I couldn't do exactly what you described because `criStatea` is an array and the `Apply` didn't like that. I did it, a bit differently though.

Reply:
Yeah, I understand, I was not completely sure you could do it that way, but anyway it looks much better in my view...